src/coreclr/vm/frames.h

```diff
@@ -487,7 +487,7 @@ class Frame : public FrameBase
     {
         WRAPPER_NO_CONTRACT;
         TADDR ptr = GetReturnAddressPtr();
-        _ASSERTE(ptr != NULL);
+        _ASSERTE(ptr != 0);
```

Comment:
`ptr` not a good name for non-pointer (TADDR expands to `unsigned long`)

Reply:
`TADDR` behaves like `void*` in most situations. It is defined as non-pointer type to make DAC magic macros and templates work, but you will find many variables of type `TADDR` with `p`, `ptr` or `pointer` in the name.

`(TADDR)NULL` is the most common pattern used for these situations. We also have `PTR_NULL` macro that is sometimes used as well. `0` looks a bit off when working with pointers.

Reply:
Can we adjust our pointer-like types to work with `nullptr`?

Reply:
```suggestion
        _ASSERTE(ptr != (TADDR)NULL);
```